python/ray/_private/test_utils.py

```diff
@@ -1366,3 +1366,11 @@ def job_hook(**kwargs):
     cmd = " ".join(kwargs["entrypoint"])
     print(f"hook intercepted: {cmd}")
     sys.exit(0)
+
+
+def wandb_setup_api_key_hook():
```

Comment:
hmm, is it used in other tests? Or should we just move it to the tune test below?